packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.8.4+8
+
+* Ensures the `result` callback and method call `arguments` send to iOS are 
+taken into consideration when picking multiple images on pre-iOS 14 devices;
```

Comment:
Nit: This is a full sentence, so should end with a period.

Reply:
Removed the first sentence.